doc/nrf/security/ap_protect.rst

```diff
@@ -173,8 +173,20 @@ Based on the available implementation types, you can configure the access port p
 Enabling AP-Protect with :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK`
 ====================================================================
 
-Setting the :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK` Kconfig option to ``y`` and compiling the firmware is enough to enable the access port protection mechanism for SoCs of the nRF53 Series and those SoCs of the nRF52 Series that feature the hardware and software type of AP-Protect.
-The access port protection configured in this way cannot be disabled without erasing the flash.
+Setting the :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK` Kconfig option to ``y`` and compiling the firmware enables the access protection mechanism for SoCs of the nRF53 Series and those SoCs of the nRF52 Series that feature the hardware and software type of AP-Protect.
+
+Enabling the Kconfig option writes the debugger register in the ``SystemInit()`` function to lock the access port protection in every boot.
+In addition to this, the UICR.APPROTECT register should be written as instructed in :ref:`app_approtect_uicr_approtect`.
+
+.. note::
+    For multi-image builds, this Kconfig option needs to be set for the first image (usually a bootloader).
+    Otherwise, the AP-Protect will not be sufficient and the debugger can be attached to the device.
```

Comment:
Why don't we just advise to use the newly-introduced sysbuild Kconfig options?

Reply:
We do it in the next sentence.

Reply:
Yes, we are still supporting the child-image builds as well.